fs-ansible/roles/lustre/tasks/format.yaml

```diff
@@ -56,6 +63,13 @@
          state: present
       loop: "{{ osts.keys() }}"
 
+    - name: Wait for OST partition to appear in /dev
+      wait_for:
+        path: "/dev/{{ item.key }}p2"
+        timeout: 30
+        sleep: 5
+      loop: "{{ osts|dict2items }}"
```

Comment:
as above, more consistent would be osts.keys() and {{ item }}

Reply:
Done